### PR TITLE
Fix dependabot comment to be added only on PR open

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Comment by xstefank** [comment]
@The-Huginn review please

File: src/main/java/io/xstefank/wildfly/bot/PullRequestFormatProcessor.java [hunk 1]
```diff
@@ -89,6 +76,22 @@ void onPullRequestEdited(@PullRequest.Edited @PullRequest.Opened @PullRequest.Sy
 
     }
 
+    void dependabotInfo(@PullRequest.Opened GHEventPayload.PullRequest pullRequestPayload,
```
Comment (petrberan) [inline src/main/java/io/xstefank/wildfly/bot/PullRequestFormatProcessor.java:79]: Isn't the java naming convention that method names should be verbs?

**Comment by petrberan** [comment]
I'm missing a test when the config contains ProjectKey other than WFLY, such as WFCORE. And a rebase

**Comment by xstefank** [comment]
I don't think we need to test multiple times that projectKey config is processed, we have a different test that solely tests projectKey

**Review by petrberan — APPROVED** [review]

**Pull request merged**